### PR TITLE
Refactor/reform

### PR DESCRIPTION
**Pull request by dcangulo**
@kencruz07 @juliusb0205

### CODE REVIEW

**Comment by dcangulo** [comment]
@kencruz07

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,24 +1,23 @@
 class UsersController < ApplicationController
   layout :layouts
-  def register
-
-  end
+  before_action :register_form, :only => [:register, :register_process]
```
Comment (kencruz07) [inline app/controllers/users_controller.rb]: Rename methods with instances of "register" to "registration"

File: app/forms/register_form.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class RegisterForm < Reform::Form
+  model :user
```
Comment (kencruz07) [inline app/forms/register_form.rb]: Add line breaks between line with different uses. Separate models, properties, validations, etc.

File: app/models/comment.rb [hunk 3]
```diff
@@ -1,4 +1,7 @@
 class Comment < ApplicationRecord
   belongs_to :post
   belongs_to :user
+  scope :latest, -> {
```
Comment (kencruz07) [inline app/models/comment.rb]: Add line break before scope, fit scope in one line

File: app/models/post.rb [hunk 4]
```diff
@@ -2,4 +2,7 @@ class Post < ApplicationRecord
   belongs_to :user
   has_many :comments
   has_and_belongs_to_many :users
+  scope :latest, -> {
```
Comment (kencruz07) [inline app/models/post.rb]: Add line break before scope, fit scope in one line

**Comment by dcangulo** [comment]
@kencruz07

**Pull request merged**